src/coreclr/jit/codegenarm64.cpp

```diff
@@ -4865,12 +4864,11 @@ void CodeGen::genCodeForJumpCompare(GenTreeOpCC* tree)
 
     if (tree->OperIs(GT_JTEST))
     {
-        ssize_t compareImm = op2->AsIntCon()->IconValue();
-
-        assert(isPow2(((size_t)compareImm)));
+        uint64_t compareImm = op2->AsIntCon()->IntegralValueUnsigned();
```

Comment:
this could be a perf regression cause... if `uint64_t` isn't treated the same as `ssize_t`